docs/docs/self-host/config.mdx

```diff
@@ -68,11 +68,9 @@ segments:
   visitors:
     name: Visitors in the US
     datasource: warehouse
-    owner: Sample User
     sql: |-
-      SELECT userid as user_id, timestamp as date, country
-      FROM pages
-      WHERE country='US'
+      SELECT userid as user_id, timestamp as date
+      FROM users
```

Comment:
You want to keep the `WHERE country='US'` part.  Otherwise, it doesn't make sense as a segment.